extensions/GradioNotebook/gradionotebook/frontend/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "gradio_notebook",
+  "version": "0.2.5",
+  "description": "Gradio UI packages",
+  "type": "module",
+  "author": "",
+  "license": "ISC",
+  "private": false,
+  "main_changeset": true,
+  "exports": {
+    ".": "./Index.svelte",
+    "./example": "./Example.svelte",
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "build": "node config/index.js --mode build",
+    "dev": "node config/index.js --mode dev"
+  },
+  "dependencies": {
+    "@datadog/browser-logs": "^5.8.0",
+    "@gradio/atoms": "0.3.0",
+    "@gradio/client": "^0.10.1",
+    "@gradio/statustracker": "0.4.0",
+    "@gradio/utils": "0.2.0",
+    "@lastmileai/aiconfig-editor": "^0.1.26",
```

Comment:
Just a reminder to ensure this matches the version in the gradio workbook repo if we bump again